src/errors.js

```diff
@@ -0,0 +1,37 @@
+export class IncogniaAPIError extends Error {
+  constructor(message, options) {
+    super(message)
+    this.name = 'IncogniaAPIError'
+    this.statusCode = options && options.statusCode
+    this.payload = options && options.payload
+  }
+}
+
+export class IncogniaError extends Error {
+  constructor(message) {
+    super(message)
+    this.name = 'IncogniaError'
+  }
+}
+
+export function handleRequestError(e) {
```

Comment:
Since this funcion throws custom error, what do you think about calling this `throwRequestError` or `throwCustomRequestError`?

Reply:
Agreed! d4e46bcfbddcac62179957a8173ff46c128b8fab